### PR TITLE
Add FAQ entries: vac_cert_booster_info, vac_cert_booster_cwa and warn_for_others

### PR DESCRIPTION
**Pull request by larswmh**
This pull request adds the mentioned faq entries in these issues: [vac_cert_booster_info + vac_cert_booster_cwa](https://github.com/corona-warn-app/cwa-website/issues/1649), [warn_for_others](https://github.com/corona-warn-app/cwa-website/issues/1695)

vac_cert_booster_info: 
DE:
chrome_bppoCIrRzZ
EN:
chrome_ujltQv5wuL

vac_cert_booster_cwa:
DE:
chrome_pmwMFWgQzy
EN:
chrome_SC37dsrWqC

warn_for_others:
DE:
chrome_udTTdhLMlbchrome_2N7OkyU7SR
EN:
chrome_hjfcTQ7q79chrome_A0b8PH1Iuu

### CODE REVIEW

File: src/data/faq.json [hunk 1]
```diff
@@ -324,6 +324,26 @@
...
                         ]
                     },
+                    {
+                        "title": "How do I become aware of booster vaccinations in the context of Corona?",
+                        "anchor": "vac_cert_booster_info",
+                        "active": true,
+                        "textblock": [
+                            "The Conference of Health Ministers has now passed a regulation on COVID 19 booster vaccinations. An additional booster vaccination is designed to help raise neutralizing antibody titers again, which may decline over time after basic immunization. This becomes particularly important if a weak/muted vaccine response is assumed or recognized after basic immunization in certain groups of vulnerable people.",
+                            "The Corona Warn App (CWA) will notify the user to consider a booster vaccination if certain conditions are met. These conditions are based on the recommendations of the Paul Ehrlich Institute and the Permanent Vaccination Commission (STIKO) of the Robert Koch Institute (RKI).",
```
Comment (MikeMcC399) [inline src/data/faq.json:333]:
```suggestion
                            "The Corona-Warn-App (CWA) will notify the user to consider a booster vaccination if certain conditions are met. These conditions are based on the recommendations of the Paul Ehrlich Institute and the Permanent Vaccination Commission (STIKO) of the Robert Koch Institute (RKI).",
```
Corona-Warn-App uses hyphens also in English - see https://www.coronawarn.app/en/

Comment (larswmh) [reply src/data/faq.json]: Committed your suggestion, thanks!

File: src/data/faq.json [hunk 2]
```diff
@@ -324,6 +324,26 @@
...
+                            "How long someone is effectively protected by a COVID 19 vaccination (basic immunization) depends on several factors and can therefore not be answered in general terms according to current knowledge. For this reason, the corresponding CWA rules for vaccination schedules and booster vaccinations are continuously reviewed and adapted in coordination with the RKI according to developments and new findings."
+                        ]
+                    },
+                    {
+                        "title": "How do I manage booster vaccination certificates in my CWA?",
+                        "anchor": "vac_cert_booster_cwa",
+                        "active": true,
+                        "textblock": [
+                            "In one of the upcoming releases, the Corona-Warn-App allows importing certificates documenting a booster vaccination.",
```
Comment (MikeMcC399) [inline src/data/faq.json:342]:
```suggestion
                            "In one of the upcoming releases, the Corona-Warn-App will allow importing certificates documenting a booster vaccination.",
```

Comment (larswmh) [reply src/data/faq.json]: Committed your suggestion, thanks!

File: src/data/faq_de.json [hunk 3]
```diff
@@ -1339,6 +1359,19 @@
...
+                            "<ul><li>Erhalten Sie im Nachhinein Kenntnis von der Gegenwart einer positiv getesteten Person auf Ihrer Veranstaltung, und hat das Gesundheitsamt Sie als Organisator einer Veranstaltung noch nicht kontaktiert, setzen Sie sich mit dem Gesundheitsamt in Verbindung. Das Gesundheitsamt kann Ihnen gegebenenfalls eine TAN ausstellen.</li><li>Klicken Sie auf der Startseite der CWA auf „QR-Code erstellen“.</li><li>Klicken Sie auf der „Meine QR-Codes“-Seite oben rechts auf das Drei-Punkte-Menü und dann auf den Menüpunkt „In Vertretung warnen“.</li><li>Klicken Sie auf dem Infoscreen auf „Weiter“.</li><li>Wählen Sie eine Veranstaltung, für welches Sie in Vertretung einer infizierten Person andere warnen möchten. Oder stellen Sie eine gelöschte Veranstaltung wieder her, indem sie den QR-Code der Veranstaltung erneut einscannen. Klicken Sie auf „Weiter“.</li><li>Geben Sie die Aufenthaltsdauer der positiv getesteten Person an, für die gewarnt werden soll. Diese Information erhalten Sie vom Gesundheitsamt. Klicken Sie auf „Weiter“.</li><li>Geben Sie die 10-stellige TAN ein, die Ihnen mitgeteilt wurde. Klicken Sie auf „Warnung abschicken“.</li></ul>",
```
Comment (MikeMcC399) [inline src/data/faq_de.json:1371]: in
"indem sie den QR-Code der Veranstaltung erneut einscannen." 
"sie" => "Sie" (capitalize)

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Minor comments regarding spelling and grammar.

**Comment by larswmh** [comment]
Thanks for your comments @MikeMcC399! Your suggested changes have been implemented

File: src/data/faq.json [hunk 4]
```diff
@@ -1330,6 +1350,19 @@
...
                         ]
+                    },
+                    {
+                        "title": "How can I warn for others?",
+                        "anchor": "warn_for_others",
+                        "active": true,
+                        "textblock": [
+                            "As an organizer of an event, starting with version 2.9, you can warn for others, such as a a person who has tested positive and was attending an event but not checked in via the Corona-Warn-App (CWA), upon request by the health department.",
```
Comment (MikeMcC399) [inline src/data/faq.json:1359]:
```suggestion
                            "As an organizer of an event, starting with version 2.9, you can warn for others, such as for a person who has tested positive and was attending an event but not checked in via the Corona-Warn-App (CWA), upon request by the health department.",
```

Comment (larswmh) [reply src/data/faq.json]: Committed your suggestion, thanks!

**Comment by dsarkar** [comment]
@MikeMcC399 @larswmh Thanks!

**Pull request merged**